internal/core/application/operator_service.go

```diff
@@ -1315,7 +1315,40 @@ func (o *operatorService) MarketFragmenterSplitFunds(
 		return
 	}
 
-	o.splitMarketFragmenterFunds(ctx, mkt, utxos, int(millisatsPerByte), chRes)
+	chRes <- FragmenterSplitFundsReply{
+		Msg: "fetching fee account funds",
+	}
+	feeUtxos, err := o.getAllUnspentsForAccount(ctx, domain.FeeAccount)
+	if err != nil {
+		errMsg := fmt.Errorf(
+			"error while fetching fee account funds: %s", err,
+		)
+		if err == domain.ErrVaultAccountNotFound {
+			errMsg = fmt.Errorf(
+				"no funds detected for fee account.\n" +
+					"You need to deposit some LBTC funds to this account used to " +
+					"pay for network fees of transactions",
+			)
+		}
+		chRes <- FragmenterSplitFundsReply{
+			Err: errMsg,
+		}
+		return
+	}
+	if len(feeUtxos) <= 0 {
```

Comment:
maybe checking the dust limit here, but maybe much  better to use the fee treshold usually a split can be 30k sats

Reply:
We check for fee account low balance at every deposit, withdraw or trade transaction.

Here is enough to make sure that the fee account owns at least one utxo because in case the fee account balance is too low, the user should have already been notified (warn on stdout, webhook if registered, and we do not allow amount under 5k sats in fee fragmenter).
Eventually, the fragmentation will return error when crafting the transaction because the of the failing coin selection.